lib/resin-lint.ts

```diff
@@ -1,10 +1,10 @@
 import * as Promise from 'bluebird';
-import * as coffeelint from 'coffeelint';
-import * as reporter from 'coffeelint/lib/reporters/default';
+const coffeelint: any = require('coffeelint');
+const reporter: any = require('coffeelint/lib/reporters/default');
 import * as depcheck from 'depcheck';
 import * as fs from 'fs';
 import * as glob from 'glob';
-import * as merge from 'merge';
+const merge: any = require('merge');
```

Comment:
If I was being really picky, I'd rather these `: any` imports were typed.

Reply:
I completely agree, I just want to get this thing working asap - I didn't realise it had been merged in a broken state way back (and just never published)